motif-docs/pages/guides/config/using-code.mdx

```diff
@@ -13,10 +13,34 @@ import dashBoardPic from '../../../../public/dashboard-maintenance-window.png'
 
 To use your own file, you'll need to create a compatible JSON, YAML, or CUE file and add it to the path you indicated in the configuration.
 
-- Each git repo can have multiple `flightcontrol.json` files
-- Each `flightcontrol.json` corresponds to a single `Project` (as shown in the Flightcontrol dashboard)
-- Each `Project` can have multiple `Environment`'s
-- Each `Environment` can have multiple `Service`'s
+## How do `flightcontrol.json` files fit into Flightcontrol concepts?
+- Each git repo can have multiple `flightcontrol.json` files which corresponds to a single `Project`
+- A `Project` can have multiple `Environments`
+- An `Environment` can have multiple `Service`'s
+
+<table style="width:100%">
+  <tr>
+    <th colspan="16">GitHub</th>
+  </tr>
+  <tr>
+    <td colspan="8">flightcontrol.json</td>
+    <td colspan="8">flightcontrol.json</td>
+  </tr>
+  <tr>
+    <td colspan="8">Project</td>
+    <td colspan="8">Project</td>
+  </tr>
+  <tr>
+    <td colspan="2">Environment</td>
+    <td colspan="2">Environment</td>
+    <td colspan="6"> </td>
+  </tr>
+  <tr>
+    <td>Service</td>
+    <td>Service</td>
+    <td colspan="14"> </td>
+  </tr>
+</table>
```

Comment:
can you use a markdown table instead?

Also, if I understand your intent, I think we can have `flightcontrol.json` as only one cell (because it's for both projects) and then change `Project` and `Project` to be `Project A` and `Project B` and same for Service.

Reply:
@flybayer Sounds good on assigning letters to`Project` s etc. 

I had two `flightcontrol.json` cells to indicate that each github repo can have multiple json files as indicated on line 16 in the current file and each one corresponds to a project. Would it work to have `flightcontrol.json A` and then `Project A`, and then same for B? 

I went with HTML bc GitHub Markdown doesn't have a colspan option that I can find. If it needs to be markdown, we might reconsider how to illustrate this.

Reply:
ok yes, but name them differently, so like `flightcontrolA.json` and `flightcontrolB.json` or have different paths like `a/flightcontrol.json` and `b/flightcontrol.json`